Katello/hammer-cli-katello
# Fixes #37230 - Remove SCA hammer references

* What are the changes introduced in this pull request?
  * Removing the SCA command in hammer to disable/enable/check SCA status
  * Removing SCA references out of organization commands
  * Removed SCA command tests
  * Updated organization tests to remove SCA unit test

* Considerations taken when implementing this change?
  * To make sure nothing breaks

* What are the testing steps for this pull request?
  * Make sure you have the latest Katello master changes pulled in with the API updates around SCA
  * Check out PR
  * Try to view info on an org and make sure you don't see any SCA items
  * Verify the `simple-content-access` sub command is missing
